src/LookingGlassConfig.ts

```diff
@@ -321,31 +336,80 @@ export class LookingGlassConfig extends EventTarget {
 		this.updateViewControls({ inlineView: v })
 	}
 
+	get capturing() {
+		return this._viewControls.capturing
+	}
+
+	set capturing(v : boolean) {
+		this.updateViewControls({ capturing: v })
+	}
+
+	get popup() {
+		return this._viewControls.popup
+	}
+
+	set popup(v : Window | null) {
+		this.updateViewControls({ popup: v })
+	}
+
+	get XRSession() {
+		return this._viewControls.XRSession
+	}
+
+	set XRSession(v) {
+		this.updateViewControls({XRSession: v})
+	}
 	// Computed
 
 	public get aspect() {
-		return this._calibration.screenW.value / this._calibration.screenH.value
+		return (this._calibration.screenW.value / this._calibration.screenH.value)
 	}
 
 	public get tileWidth() {
-		return Math.round(this.tileHeight * this.aspect)
+		return Math.round(quiltResolution / this.quiltWidth)
```

Comment:
Instead of configuring the quilt tile width from the tiltHeight value, calculate it based on the quiltResolution divided by the `quiltWidth` value. 

The terms `quiltWidth` and `quiltHeight` refer to the columns and rows. These terms should be renamed in the follow up 0.5.0 release for consistency with other Looking Glass software. I omitted them from being changed in this PR, however this can be reconsidered.